tests/main/views/test_marketplace.py

```diff
@@ -864,10 +864,10 @@ def test_catalogue_of_briefs_page(self):
         ss_elem = document.xpath("//p[@class='search-summary']")[0]
         assert self._normalize_whitespace(self._squashed_element_text(ss_elem)) == "2 opportunities"
 
-        dos1_framework_label = document.xpath("//div[@class='search-result']/ul[2]/li[2]/text()")[0].strip()
-        dos2_framework_label = document.xpath("//div[@class='search-result']/ul[2]/li[2]/text()")[1].strip()
-        assert dos1_framework_label == "Digital Outcomes and Specialists"
-        assert dos2_framework_label == "Digital Outcomes and Specialists 2"
+        first_brief_specialist_role = document.xpath("//div[@class='search-result']/ul[2]/li[2]/text()")[0].strip()
+        second_brief_specialist_role = document.xpath("//div[@class='search-result']/ul[2]/li[2]/text()")[1].strip()
+        assert first_brief_specialist_role == "Content Specialist"
+        assert second_brief_specialist_role == "Content Specialist 2"
```

Comment:
These will need updating based on my comments regarding fixtures not always having a specialist role and also un-camelcasing it.